backend/addcorpus/python_corpora/save_corpus.py

```diff
@@ -87,17 +87,26 @@ def _save_field_in_database(field_definition: FieldDefinition, configuration: Co
     field.full_clean()
     return field
 
-def _set_corpus_inactive(corpus_name):
+def _deactivate(corpus):
+    corpus.active = False
+    corpus.save()
+
+def _set_active(corpus):
```

Comment:
These method names don't match. 
```suggestion
def _deactivate(corpus):
    corpus.active = False
    corpus.save()

def _activate(corpus):
```